### PR TITLE
Propose a solution to the "Remove Chars" problem

### PR DESCRIPTION
**Pull request by neuodev**

### CODE REVIEW

File: problem/remove-characters_en.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
...
 
-There is no solution yet.
+## Problem analysis
+This problem is a classic example of a `stack` problem as you need to remember the last `char` you have while you are going through the string. 
 
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/problem/remove-characters_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+## Solution
+
+```ts
+  let stack: string[] = []
```
Comment (BFEdev) [inline problem/remove-characters_en.md]: shall we use const?

File: problem/remove-characters_en.md [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+  for (let i = 0; i < input.length; i++) {
+    let char = input[i]
```
Comment (BFEdev) [inline problem/remove-characters_en.md]: shall we use const?

File: problem/remove-characters_en.md [hunk 3]
```diff
@@ -1,4 +1,39 @@
...
+**Output:** `caa`
+
+1. At first the stack will be empty. 
+2. The two letters `ca` are not `b` so they will be added to stack `[c, a]`. 
+3. The next `bb` will not be added to the stack. Stack still `[c, a]`
+4. The next `aa` will be added to the stack as they are not `b`. Stack will be `[c, a, a, a]`
+5. The next `b` wil not be added to the stack. Stack still `[c, a, a, a]`
+6. Becuase the next char is `c` and the last element added to the stack was `a`, `a` will be removed and `c` will not be added. Stack will be `[c, a, a]`
+7. The same will apply for the next `c`. Stack will be `[c, a]`
+8. Finaly, the last `a` in the input will be added to the stack. Stack will be `[c, a, a]` (**our output**)
```
Comment (BFEdev) [inline problem/remove-characters_en.md]: it is a bit verbose here, shall we  align them  better? for example using a table

| char | stack | explanation|
|-----|-----|-----|
| c | [] | init|
| a | ['c'] | xxx
| b | ['c', 'a'] | xxx

**Review by BFEdev — CHANGES_REQUESTED** [review]
Thanks for your contribution. Just one more step to get merged

File: problem/remove-characters_en.md (hunk 1 above)
Comment (neuodev) [reply problem/remove-characters_en.md]: yup

**Pull request merged**